corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -388,6 +388,21 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.templateType = 'select';
         self.choices = question.choices;
         self.isMulti = true;
+        self.hideLabel = ko.observable(options.hideLabel);
```

Comment:
I don't think this needs to be an observable - its value is never going to change, right?

Reply:
It was an observable in `ChoiceLabelEntry` so I just copied it. But I'm going to change it because it makes the code in `MultiSelectEntry' much cleaner in computing `colStyle` as well

Reply:
You'll notice I removed the observable in ChoiceLabelEntry as well